content/attacks/posts/2023-01-12-Lendhub.md

```diff
@@ -22,11 +24,13 @@ Lendhub lost approximately $6 million.
 
 ## Timeline
 
-- **January 12, 2023:**  Lendhub is hacked
-- **January 13, 2023:**  [Lendhub announced the hack on their platform and the extent of the losses, with around $6 million stolen](https://twitter.com/LendHubDefi/status/1613846541651030018)
-- **January 13, 2023:**  [The attacker has made 11 transactions totaling 1,100 ETH to TornadoCash](https://twitter.com/SlowMist_Team/status/1613906600279900162)
-- **February 26, 2023:**  [The attacker has deposited another 2,415.4 ETH into Tornado.cash](https://twitter.com/PeckShieldAlert/status/1630027178426642432)
+- **January 11, 2023, 10:11:35 PM +UTC:** [The exploiter received 100 ETH from Tornado.cash.](https://etherscan.io/tx/0xdc560bbaad3972ea3ee012b1cd6829364631b59f30115aa3f78e761818646524)
+- **January 11, 2023, 10:38:59 PM +UTC:** [The exploiter swapped ETH for Heco Token (HT) and USDT and transferred tokens cross-chain to the HECO chain.](https://www.hecoinfo.com/en-us/tx/0x17e47fb60e37ccdbb93394bc82b25ba213b936f7123bc5d4a4a16b043dfacb39)
+- **January 12, 2023:** The exploiter then interacted with the attack contracts multiple times, which drained assets from the new Lendhub HECO market.
+- **January 13, 2023:** [Lendhub announced the hack on their platform and the extent of the losses, with around $6 million stolen](https://twitter.com/LendHubDefi/status/1613846541651030018)
+- **January 13, 2023:** [The attacker has made 11 transactions totaling 1,100 ETH to Tornado.cash](https://twitter.com/SlowMist_Team/status/1613906600279900162)
+- **February 26, 2023:** [The attacker has deposited another 2,415.4 ETH into Tornado.cash](https://twitter.com/PeckShieldAlert/status/1630027178426642432)
 
 ## Security Failure Causes
 
-The main reason for the Lendhub attack was the failure to remove a deprecated token from the market properly. This vulnerability allowed attackers to exploit the protocol's smart contracts and steal funds.
+- **Old and new IBSV tokens exist simultaneously:** This made it possible to mint and redeem tokens on the old market, while simultaneously borrowing money against them on the new market
```

Comment:
The bullet point should be common cause, I would suggest `Dev Team Negligence`. And mention that main reason was overlooked deprecated token from the market, in the description itself.